src/OrdersList/OrdersListSearchConfig.js

```diff
@@ -19,31 +21,44 @@ const customSearchMap = {
   'dateOrdered': searchByDate,
 };
 
-function getCqlQuery(query, qindex, dateFormat) {
-  return customSearchMap[qindex]?.(query, dateFormat) || `*${query}*`;
+function getCqlQuery(query, qindex, dateFormat, customField) {
+  return customField?.type === 'DATE_PICKER'
+    ? searchByDate(query, dateFormat)
+    : customSearchMap[qindex]?.(query, dateFormat) || `*${query}*`;
 }
 
-const getKeywordQuery = (query, dateFormat) => indexes.reduce(
-  (acc, sIndex) => {
-    const cqlQuery = getCqlQuery(query, sIndex, dateFormat);
+const getKeywordQuery = (query, dateFormat, customFields) => {
+  const customFieldIndexes = (customFields || [])
+    .filter((cf) => ['DATE_PICKER', 'TEXTBOX_SHORT', 'TEXTBOX_LONG'].includes(cf.type))
```

Comment:
I think it would be better if you could extract `'DATE_PICKER', 'TEXTBOX_SHORT', 'TEXTBOX_LONG'` and other similar values to `constants` similar to `FILTERS` object.

Reply:
I concur with the previous comment. Kindly store these values in a constant that bears a descriptive name.